bombastic/api/src/server.rs

```diff
@@ -73,15 +75,18 @@ enum Error {
     InvalidContentEncoding,
 }
 
+#[allow(clippy::unwrap_used)]
+fn accept_encodings() -> Vec<QualityItem<Preference<Encoding>>> {
+    ACCEPT_ENCODINGS.iter().map(|s| s.parse().unwrap()).collect()
+}
+
 impl error::ResponseError for Error {
     fn error_response(&self) -> HttpResponse {
         let mut res = HttpResponse::build(self.status_code());
         res.insert_header(ContentType::plaintext());
         match self {
             Self::InvalidContentType => res.insert_header(Accept::json()),
-            Self::InvalidContentEncoding => res.insert_header(AcceptEncoding(
-                ACCEPT_ENCODINGS.iter().map(|s| s.parse().unwrap()).collect(),
```

Comment:
Personally, I'm ok with the `unwrap` here. It's a non-public const. To me, as someone familiar with HTTP headers and their string values, the code is more clear without dragging in the `QualityItem`, `Preference`, and `Encoding` abstractions only to add an `allow` directive.

In the interest of clarity, I'd probably `s/unwrap/expect("unexpected")/` just to appease clippy.

Reply:
The idea was to isolate that part into its own function. Changed it back.